sim/deathknight/death_strike.go

```diff
@@ -11,9 +11,9 @@ var DeathStrikeActionID = core.ActionID{SpellID: 49924}
 
 func (dk *Deathknight) newDeathStrikeSpell(isMH bool, onhit func(sim *core.Simulation, spell *core.Spell, spellEffect *core.SpellEffect)) *RuneSpell {
 	bonusBaseDamage := dk.sigilOfAwarenessBonus(dk.DeathStrike)
-	weaponBaseDamage := core.BaseDamageFuncMeleeWeapon(core.MainHand, true, 297.0+bonusBaseDamage, 1.0, 0.75, true)
+	weaponBaseDamage := core.BaseDamageFuncMeleeWeapon(core.MainHand, true, 297.0+bonusBaseDamage, 0.75, true)
 	if !isMH {
-		weaponBaseDamage = core.BaseDamageFuncMeleeWeapon(core.OffHand, true, 297.0+bonusBaseDamage, dk.nervesOfColdSteelBonus(), 0.75, true)
+		weaponBaseDamage = core.BaseDamageFuncMeleeWeapon(core.OffHand, true, 297.0+bonusBaseDamage, dk.nervesOfColdSteelBonus()*0.75, true)
```

Comment:
Pre-existing: the [sigil](https://www.wowhead.com/wotlk/spell=60837)'s bonus damage should be 420 for all strikes it applies to: Death Strike (75% * 420 = 315), Obliterate (80% * 420 = 336), and Scourge Strike (70% * 420 = 294; tooltip says 189 because 45% was the original modifier, cp. [here](https://wowpedia.fandom.com/wiki/Scourge_Strike)).

Since it also applies to the OH spells, it's quite likely they gain more damage bonus than "intended": [DS's OH strike](https://www.wowhead.com/wotlk/spell=66953) has 148 "flat damage" (compared to 297 for the MH), and the sigil might apply directly, and _then_ be multiplied by Nerves of Cold Steel (for (148 + 420) * 125% = 710 effective bonus damage).

This should be tested, though, since "Apply Aura: Modifies Effect #1's Value" effects can't be predicted reliably.

Reply:
I can fix the base damage if you think that is how it should work, but has there been any testing done or are we just going off tooltips? I don't want to be doing all these changes if all the base of your statements is wowhead tooltips... Any logs, any testing done?